drawgridbox.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+from pymol.cgo import *
+from pymol import cmd
+import itertools
+from random import randint
+
+#############################################################################
+#
+# drawgridbox.py -- Draw a grid box around a selection
+#
+# AUTHOR: Cunliang Geng
+# DATE  : 19/5/2016
+#
+# Acknowledgement:
+# This scirpt was written based on the DrawBoundingBox by Jason Vertrees
+#
+#############################################################################
+def drawgridbox(selection="(all)", nx=10, ny=10, nz=10, padding=0.0, lw=2.0, r=1.0, g=1.0, b=1.0):
+        """
+        DESCRIPTION
+                Given selection, draw a grid box around it.
+
+        USAGE:
+                drawgridbox [selection, [nx, [ny, [nz, [padding, [lw, [r, [g, b]]]]]]]]
+
+        PARAMETERS:
+                selection,              the selection to enboxen.
+                                        defaults to (all)
+
+                nx,                     number of grids on axis X
+                                        defaults to 10
+
+                ny,                     number of grids on axis Y
+                                        defaults to 10
+
+                nz,                     number of grids on axis Z
+                                        defaults to 10
+
+                padding,                defaults to 0
+
+                lw,                     line width
+                                        defaults to 2.0
+
+                r,                      red color component, valid range is [0.0, 1.0]
+                                        defaults to 1.0
+
+                g,                      green color component, valid range is [0.0, 1.0]
+                                        defaults to 1.0
+
+                b,                      blue color component, valid range is [0.0, 1.0]
+                                        defaults to 1.0
+
+        RETURNS
+                string, the name of the CGO box
+
+        NOTES
+                * This function creates a randomly named CGO grid box. The user can
+                specify the number of grids on X/Y/Z axis, the width of the lines,
+                the padding and also the color.
+        """
+
+        ([minX, minY, minZ],[maxX, maxY, maxZ]) = cmd.get_extent(selection)
```

Comment:
@CunliangGeng This line is indented using 8 * " ". Could you please reduce this to 4 and squash all your reindenting commits into one?